WcaOnRails/app/views/results_submission/edit.html.erb

```diff
@@ -2,8 +2,11 @@
 <%= render layout: 'nav' do %>
   <h1><%= yield(:title) %></h1>
 
+  <p>Enter the body of your email to the Results Team.</p>
+  <p>Make sure to include any changes to the original schedule, as well as any other additional details required by the 'Submitting competition results' section of the <a href=<%= delegate_crash_course_path %>>crash course</a>.</p>
```

Comment:
ERB has a helper for simple links like this. I think it's something like `<%= link_to "crash course", delegate_crash_course_path %>`

Reply:
Good point - fixing it

Reply:
nit: I would prefer the link text to say "delegate crash course" instead of just "crash course".